includes/wp-admin/class-settings.php

```diff
@@ -22,6 +22,21 @@ class Settings {
 	public static function init() {
 		\add_action( 'admin_init', array( self::class, 'register_settings' ), 11 );
 		\add_action( 'admin_menu', array( self::class, 'add_settings_page' ) );
+
+		\add_filter(
+			'screen_options_show_submit',
+			function ( $show_submit, $screen ) {
+				if ( 'settings_page_activitypub' !== $screen->id ) {
+					return $show_submit;
+				}
+
+				return true;
+			},
```

Comment:
If it's the same to you, let's make that a class method for consistency and so it could be unhooked if need be?